### DISCUSSION TITLE
HTML loads but without CSS, css components are in JS loaded as a module

### DISCUSSION THREAD

**Discussion by KishonShrill** [vercel/vercel#10556] [Help] [answered]
I tried to upload my site to vercel. Built and deployed but it only displays the html and no CSS.

Site:
https://website-portfolio-gvpot4af2-kishonshrill.vercel.app/

**Comment by sebalaini** [comment, marked as answer]
if you take a look at the console you will see that you are loading the CSS stylesheet with https://website-portfolio-gvpot4af2-kishonshrill.vercel.app/website-portfolio/assets/dark-mode-eecc09e8.css while the correct path is https://website-portfolio-gvpot4af2-kishonshrill.vercel.app/assets/dark-mode-eecc09e8.css

**Reply by KishonShrill** [reply]
Thank you very much!
After removing "base: /website-portfolio/" in vite.config.js it worked properly after the change... Bu the question is, why does that not work? Why does adding a "base:" not connect? Is this a problem while building in vercel?

**Comment by sebalaini** [comment]
I guess you are referring to this https://vite-plugin-ssr.com/base-url I haven't deployed any Vite app but if you are setting `"base: /website-portfolio/"` by reading right now the doc it says that your new root is now `/website-portfolio/` instead of just your URL
